### PR TITLE
add sections to header

### PR DESCRIPTION
**Pull request by eduramme**

### CODE REVIEW

File: pages/domains/index.tsx [hunk 1]
```diff
@@ -34,39 +34,39 @@ export default function RegisterNamePage() {
...
-                  }}
-                </ConnectButton.Custom>
-              </div>
+        {!address ? (
+          <div className="w-full flex flex-col gap-4 items-center justify-center">
+            <Heading level="2" className="text-start text-[26px]">
+              You have to connect your wallet to see your domains 😉
+            </Heading>
+            <div>
+              <ConnectButton.Custom>
```
Comment (FrancoAguzzi) [inline pages/domains/index.tsx:43]: We are not anymore using ConnectButton, please search for ConnectWallet component or something like, we are using a button to connect Metamask only

**Review by FrancoAguzzi — CHANGES_REQUESTED** [review]

File: components/01-atoms/DappHeader.tsx [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+    href: "/domains",
+  },
+];
 
 export const DappHeader = () => {
   const { address, chain } = useAccount();
   const { switchChain } = useSwitchChain();
   const [isClient, setIsClient] = useState(false);
+  const router = useRouter();
+  const currentRoute = router.pathname;
```
[the remark this answers is not in the document]
Comment (eduramme) [reply components/01-atoms/DappHeader.tsx:36]: yup

[the remark this answers is not in the document]
Comment (FrancoAguzzi) [reply components/01-atoms/DappHeader.tsx:36]: Where you able to test it?

[the remark this answers is not in the document]
Comment (eduramme) [reply components/01-atoms/DappHeader.tsx:36]: @FrancoAguzzi Yes, I tested it

**Review by FrancoAguzzi — APPROVED** [review]

File: components/01-atoms/DappHeader.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (FrancoAguzzi) [reply components/01-atoms/DappHeader.tsx:36]: Perfect

**Pull request merged**